app/javascript/react/screens/App/Plan/components/PlanRequestDetailList/PlanRequestDetailList.js

```diff
@@ -432,15 +482,20 @@ class PlanRequestDetailList extends React.Component {
                         <span id="downloadLogInProgress">{__('Download log in progress...')}</span>
                       </label>
                     ) : (
-                      <a
-                        href="#"
-                        onClick={e => {
-                          e.preventDefault();
-                          downloadLogAction(task);
-                        }}
+                      <DropdownButton
```

Comment:
Ok quick question.. I have no idea how this worked before... but If a user doesn't use a pre/post migration playbook, will they still have a log to download here? If so... maybe there should be a third option shown... to download that...

(before we had playbook support there were logs to download, just want to make sure we don't lose those)

Reply:
ahh see thats what I meant! 😏 🤣